runtime/tests/integration_test.rs

```diff
@@ -364,6 +371,91 @@ fn reward_block_authors() {
 		});
 }
 
+#[test]
+fn initialize_crowdloan_addresses_with_batch_and_pay() {
+	ExtBuilder::default()
+		.with_balances(vec![
+			(AccountId::from(ALICE), 2_000 * GLMR),
+			(AccountId::from(BOB), 1_000 * GLMR),
+		])
+		.with_collators(vec![(AccountId::from(ALICE), 1_000 * GLMR)])
+		.build()
+		.execute_with(|| {
+			// set parachain inherent data
+			set_parachain_inherent_data();
+			set_author(AccountId::from(ALICE));
```

Comment:
Why do we need Alice as an author for this test?

Reply:
Not really, we just need an author